sql/schema.sql

```diff
@@ -34,3 +38,18 @@ ALTER TABLE ONLY repos
 ALTER TABLE ONLY issues
   ADD CONSTRAINT fk_repos FOREIGN KEY (repo_owner, repo_name) 
   REFERENCES repos (owner, name) ON DELETE CASCADE;
+
+--------------
+-- TRIGGERS --
+--------------
+
+CREATE OR REPLACE FUNCTION update_timestamp() 
```

Comment:
Yep, this is the correct way to update `updated_at` columns in PostgreSQL automatically: we need to create a trigger (custom SQL function) and call it for each row before the update. Just a minor stylistic suggestion: I would love to see `updated_at` column name in the name of this trigger, so it will be clear which one of the timestamps is updated. Because later we might have more timestamp times like when the issue was created on GitHub, what was the timestamp of the last comment, etc. So the trigger can have a name like `update_update_at` or something like that.